data/info.json

```diff
@@ -0,0 +1,194 @@
+{
+  "San Francisco County": {
+    "info": [
```

Comment:
@nikobooth @thuongho on Figma I see "Last updated dd/mm/yy" in each card. We better have it like "updated_date" : "2020-08-13".

Reply:
@kengoy Oh yea I forgot about the updated time. On the google sheet I didn't see an updated time. @nikobooth can you fill that in? It can go right before the `"info"`

Reply:
Added the line to the relevant sheets!

Also, it says on here I've been requested to review, I'm sure I don't have anything to add right now and also not sure where to indicate my approval, so I'll just take myself off of the review panel :D